packages/squiggle-lang/src/rescript/Distributions/GenericDist/GenericDist_Types.res

```diff
@@ -95,3 +97,130 @@ module Operation = {
     | Mixture(_) => `mixture`
     }
 }
+
+module Constructors = {
+  type t = Operation.genericFunctionCallInfo
+
+  module UsingDists = {
+    let mean = (dist): t => FromDist(ToFloat(#Mean), dist)
+    let sample = (dist): t => FromDist(ToFloat(#Sample), dist)
+    let cdf = (dist, f): t => FromDist(ToFloat(#Cdf(f)), dist)
+    let inv = (dist, f): t => FromDist(ToFloat(#Inv(f)), dist)
+    let pdf = (dist, f): t => FromDist(ToFloat(#Pdf(f)), dist)
```

Comment:
why is `f` the argument name? seems like `x` would read more naturally.

Reply:
f = float. x would also be fine, could change

Reply:
`f` just evokes function, that's all. Elsewhere in the codebase the input to `pdf`, `cdf`, etc. are `x`.

Reply:
Good point, will change